scripts/performance/perf_load/perf_processes.py

```diff
@@ -101,12 +102,15 @@
 parser.add_argument('--taa_version', default="test_taa", type=str, required=False,
                     help='Transaction author agreement version')
 
+parser.add_argument('--promotion_shift', default=0, type=int, required=False,
```

Comment:
Should it be `0` be default? I think something like 60 sec is a better default value.

Reply:
Done.